area/midgaard.are

```diff
@@ -174,19 +174,27 @@ AFN ABCD 0 0
 stand stand female 4500
 0 0 medium 0
 #3011
-frank bruno~
-Frank Bruno~
-Frank Bruno stands here protecting Xania.
+Concordius~
```

Comment:
@mattgodbolt This is a first pass at the new guardian. There are a couple more things I want to do like add new behavior when he's fighting, and add some life to the "pet" eagle, Aquila.

Reply:
Not doing those things yet so that I don't end up with a huge PR.

Reply:
Cheers chap